### PR TITLE
Add routing rules for `FUNCTION LIST`.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Yury-Fridlyand** [comment]
Closing - not relevant. Caused by my misunderstanding of how `FUNCTION LOAD` and `FUNCTION LIST` should work.

**Comment by Yury-Fridlyand** [comment]
Reopening - we still need routing update to a random node for `FUNCTION LIST`.

**Review by aaron-congo — APPROVED** [review]

File: redis/src/cluster_routing.rs [hunk 1]
```diff
@@ -453,6 +453,7 @@ fn base_routing(cmd: &[u8]) -> RouteBy {
         | b"CONFIG GET"
         | b"DEBUG"
         | b"ECHO"
+        | b"FUNTION LIST"
```
Comment (barshaul) [inline redis/src/cluster_routing.rs:456]: FUNTION => FUNCTION

Comment (Yury-Fridlyand) [reply redis/src/cluster_routing.rs]: Fixed. Shame on me!

**Review by barshaul — APPROVED** [review]

**Pull request merged**